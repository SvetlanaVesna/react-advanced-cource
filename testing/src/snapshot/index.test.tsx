import React from 'react'
import { NameProvider } from '../rtl/react-context'
import renderer from 'react-test-renderer'

describe('snapshot test NameProvider', () => {
  test('toMatchSnapshot', () => {
    const tree = renderer.create(
      <NameProvider first="first" last="last">
        <p>ololo</p>
      </NameProvider>,
    )
    expect(tree).toMatchSnapshot()
  })

  test('renders correctly', () => {
    const tree = renderer.create(<a href="http://www.facebook.com">Facebook</a>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
