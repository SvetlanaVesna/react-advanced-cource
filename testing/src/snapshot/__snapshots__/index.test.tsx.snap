// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test NameProvider renders correctly 1`] = `
<a
  href="http://www.facebook.com"
>
  Facebook
</a>
`;

exports[`snapshot test NameProvider toMatchSnapshot 1`] = `
<p>
  ololo
</p>
`;
